docs/fit/simultaneousfits.rst

```diff
@@ -0,0 +1,146 @@
+.. _fit_simultaneous:
+
+Simultaneous fits
+=================
+
+Sherpa uses the :py:class:`~sherpa.data.DataSimulFit` and
+:py:class:`~sherpa.models.model.SimulFitModel`
+classes to fit multiple datasets and models simultaneously.
+This can be done explicitly, by combining individual data sets
+and models into instances of these classes, or implicitly
+with the :py:meth:`~sherpa.fit.Fit.simulfit` method.
+
+It only makes sense to perform a simultaneous fit if there is
+some "link" between the various data sets or models, such
+as sharing one or more model components or
+:ref:`linking model parameters <params-link>`.
+
+Example
+-------
+This example replicates fitting a few forbidden emission lines
+in an echelle spectrum where one line shows up in more
+than one order. The spectrum is noisy, so we want to use all
+the available data to constrain the fit. In reality, we would
+load the data from a file, but for the purpose of this example
+we will simulate two orders. One of them contains just the
+[Si II] line at 6716 Ang, while the other one contains both this
+line and the second member of the [Si II] doublet at 6731 Ang.
+
+.. plot::
+    :include-source:
+    :context:
+    :nofigs:
+
+    >>> import numpy as np
+    >>> from sherpa.models import Gauss1D
+    >>> wave_order26 = np.arange(6686, 6720, 1.1)
+    >>> wave_order27 = np.arange(6705, 6735, 1.2)
+    >>> rng = np.random.default_rng()
+    >>> # Make two components to simulate the lines
+    >>> line1 = Gauss1D('line1')
+    >>> line1.pos = 6716
+    >>> line1.fwhm = 2
+    >>> line1.ampl = 0.5
+    >>> line2 = Gauss1D('line2')
+    >>> line2.pos = 6731
+    >>> line2.fwhm = 2
+    >>> line2.ampl = 1
+    >>> # Flux is normalized to about 1, but not exactly right
+    >>> flux_order26 = rng.normal(0.98, 0.1, size=len(wave_order26))
+    >>> flux_order26 += line1(wave_order26)
+    >>> flux_order27 = rng.normal(1.11, 0.1, size=len(wave_order27))
+    >>> flux_order27 += line1(wave_order27) + line2(wave_order27)
+
+Let's now do a quick plot of what we simulated. For real data, we would
+want to get an idea what it looks before fitting!
+
+.. plot::
+    :include-source:
+    :context:
+
+    >>> from matplotlib import pyplot as plt
+    >>> _ = plt.plot(wave_order26, flux_order26, 'o', label='order 26')
+    >>> _ = plt.plot(wave_order27, flux_order27, 'o', label='order 27')
+    >>> _ = plt.xlabel('Wavelength (Angstrom)')
+    >>> _ = plt.ylabel('Flux (normalized)')
+    >>> _ = plt.legend()
```

Comment:
The plot I see is

https://sherpa--2303.org.readthedocs.build/en/2303/_downloads/db8048269eeba0a281981c3522483a2f/simultaneousfits-2.png

<img width="768" height="694" alt="image" src="https://github.com/user-attachments/assets/5d456fd0-216e-4d2d-a099-d92e9015250c" />


which isn't informative to me. For some reason it's included data with `x ~ 0` (rather than the `x ~ 6700` values we expect).